lib/models/index.js

```diff
@@ -24,16 +24,28 @@ sequelize.authenticate()
 
 
 let db = {}
+let modelNames = []
 
+// Get all the database table files from the files in this directory
 fs.readdirSync(__dirname)
 	.filter(file => {
 		return (file.indexOf('.') !== 0) && (file !=='index.js')
 	})
 	.forEach(file => {
 		let model = sequelize.import(path.join(__dirname, file))
 		db[model.name] = model
+		model.sync({force: false})
+		modelNames.push(model.name)
 	})
 
+
+modelNames.forEach(modelName => {
+	if (db[modelName].associate) { // if this function exists
+		db[modelName].associate(db) // call it
+		db[modelName].sync({alter: true}) // and re-sync the table
```

Comment:
The docs claim that `alter: true` is [not recommended for production use](http://docs.sequelizejs.com/class/lib/sequelize.js~Sequelize.html#instance-method-sync); perhaps we should do this only in development?  I don't see the schema changing that much once we're done.

I can take a deeper look at this once I'm able to hack on this next week.  Ideally, we would run migrations before syncing the DB, and thus avoid the need for `alter`ation since the schema would match the latest migration after startup.  For development, though, this flag makes sense to set.